impeller/renderer/backend/gles/sampler_gles.cc

```diff
@@ -96,10 +100,12 @@ bool SamplerGLES::ConfigureBoundTexture(const TextureGLES& texture,
                    ToParam(desc.min_filter, mip_filter));
   gl.TexParameteri(target.value(), GL_TEXTURE_MAG_FILTER,
                    ToParam(desc.mag_filter));
-  gl.TexParameteri(target.value(), GL_TEXTURE_WRAP_S,
-                   ToAddressMode(desc.width_address_mode));
-  gl.TexParameteri(target.value(), GL_TEXTURE_WRAP_T,
-                   ToAddressMode(desc.height_address_mode));
+  gl.TexParameteri(
```

Comment:
How do we set the border color? `TEXTURE_BORDER_COLOR_EXT`

Reply:
Maybe this defaults to transparent black so we don't need to worry about it?

Reply:
according to https://registry.khronos.org/OpenGL-Refpages/es3/html/glTexParameter.xhtml this value defaults to transparent black so we don't need to set anything.

Reply:
I guess. I took a look at Vulkan and [setting the border color is an extension](https://registry.khronos.org/vulkan/specs/1.3-extensions/man/html/VK_EXT_custom_border_color.html). But setting it to `VK_SAMPLER_ADDRESS_MODE_CLAMP_TO_BORDER` is not. I guess this is fine. I don't think we set the color on Metal either. Just really odd.

Reply:
Flutter does not expose an API for setting the border color.

Reply:
Nevermind. Its an extension that is core in 1.1. And we do set it to [transparent black](https://github.com/flutter/engine/blob/a18c52768bbc8c779604a186dc6f9d14e73cee14/impeller/renderer/backend/vulkan/sampler_library_vk.cc#L46). Let's be explicit and do the same here as well please?

Reply:
Its is also dealers choice on Metal at the moment. Let's be explicit and set borderColor [here too](https://github.com/flutter/engine/blob/a18c52768bbc8c779604a186dc6f9d14e73cee14/impeller/renderer/backend/metal/sampler_library_mtl.mm#L25C17-L25C37). In fact, a default border color specification in `formats.h` would be good for consistency.

Reply:
> Flutter does not expose an API for setting the border color.

Yeah, just didn't want to read the spec to see if the default was transparent black. Just being explicit seemed better. There isn't any bug though.

Reply:
Issue filed: https://github.com/flutter/flutter/issues/136153